src/context.ts

```diff
@@ -47,8 +47,10 @@ export interface CacheConfiguration {
 }
 
 export interface SubgraphConfiguration {
-  subgraphKey: string;
-  mainnetSubgraphId: string;
+  subgraphKey?: string;
+  mainnetSubgraphId?: string;
+  fantomSubgraphEndpoint?: string;
+  arbitrumSubgraphEndpoint?: string;
```

Comment:
Should we also change mainnet and receive directly the whole endpoint url? since we are still in time to make breaking changes for v2

Reply:
Yeah, I guess that way people can use a hosted version if they wish to, although it will be deprecated soon. Done in 547bfccef3cc82a7388f1ff6deba396ac3ccda74